src/moduls/mantenimientos/pages/crear.vue

```diff
@@ -145,7 +148,7 @@ const crearcustomer = async () => {
 	try {
 		// Validar si el cliente ya existe
 		const clienteExistente = await buscarClienteExistente(mantenimiento.value.document, mantenimiento.value.email);
-
+		const celphone = `${mantenimiento.value.celphoneExt} ${mantenimiento.value.celphone}`.replace(/\+/g, ''); // Elimina el símbolo '+'
```

Comment:
```suggestion
		const celphone = `${mantenimiento.value.celphoneExt} ${mantenimiento.value.celphone}`;
```

Se eligió explícitamente el estándar para almacenar los números de teléfono:
`+prefijos_telefónicos_del_país número_telefónico`
`+` Porque sí básicamente, el prefijo y el número de teléfono deben estar separados por un espacio. format: UTF-8 CRLF